internal/discovery/foreign.go

```diff
@@ -79,7 +79,8 @@ func (discovery *DiscoveryCtrl) UpdateTtl(txts []*TxtData) error {
 		klog.Error(err, err.Error())
 		return err
 	}
-	for _, fc := range fcs.Items {
+	for i := range fcs.Items {
+		fc := fcs.Items[i]
```

Comment:
Are we sure that is better this kind of for loops?

Reply:
No, but gosec told me :)
https://github.com/golang/go/wiki/CommonMistakes#using-reference-to-loop-iterator-variable